frontend/src/api/api.ts

```diff
@@ -401,6 +400,30 @@ export const apiV2 = createApi({
         data: { from, to },
       }),
     }),
+    refetchTasks: build.mutation<null, null>({
```

Comment:
```suggestion
    refetchTasks: build.mutation<null, void>({
```

The parameter could be void, then there is no need to pass `null` for each call.